src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.starts__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 .starts__star:nth-child(1) {
```

Comment:
```suggestion
.stars--1 .starts__star:nth-child(1),
.stars--2 .starts__star:nth-child(-n + 2),
... ,
.stars--5 .starts__star:nth-child(-n + 5) {
```

Reply:
Consider using commas to group selectors

Reply:
Hello! I want to understand. The main problem this is because code has not commas. if i will fix it task will be done?